microsetta_public_api/server.py

```diff
@@ -32,13 +30,22 @@ def build_app(resources_config_json=None):
 if __name__ == "__main__":
     import sys
     config_fp = sys.argv[1] if len(sys.argv) > 1 else None
-    PORT = 8084
+
+    if config_fp is None:
+        resource_config = None
+        server_config = {}
+    else:
+        with open(config_fp) as fp:
+            server_config = json.load(fp)
+        resource_config = config['resources']
+
+    port = server_config.get('port', 8084)
     if config_fp:
-        app = build_app(resources_config_json=config_fp)
-        app.run(port=PORT, debug=True)
+        app = build_app(resources_config_json=resources)
```

Comment:
```suggestion
        app = build_app(resources_updates=resource_config)
```
This is my best guess